ui/src/main/java/edu/wpi/grip/ui/pipeline/source/StopStartSourceControlsView.java

```diff
@@ -0,0 +1,111 @@
+package edu.wpi.grip.ui.pipeline.source;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+import edu.wpi.grip.core.StopStartSource;
+import edu.wpi.grip.core.events.FatalErrorEvent;
+import edu.wpi.grip.core.events.SourceStartedEvent;
+import edu.wpi.grip.core.events.SourceStoppedEvent;
+import edu.wpi.grip.ui.util.DPIUtility;
+import javafx.application.Platform;
+import javafx.scene.control.ContentDisplay;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.Tooltip;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Controls for a {@link StopStartSource}
+ */
+public class StopStartSourceControlsView extends SourceControlsView<StopStartSource> {
```

Comment:
I don't see us ever reusing this class - better just make it only take a `CameraSource` instead of a `StopStartSource` so if we add more camera things (resolution settings, exposure) we can also add them to the view